manifests/complete-postgres-manifest.yaml

```diff
@@ -125,5 +125,11 @@ spec:
     certificateFile: "tls.crt"
     privateKeyFile: "tls.key"
     caFile: ""  # optionally configure Postgres with a CA certificate
+# file names can be also defined with absolute path, and will no longer be relative
+# to the "/tls/" path where the secret is being mounted by default.
 # When TLS is enabled, also set spiloFSGroup parameter above to the relevant value.
 # if unknown, set it to 103 which is the usual value in the default spilo images.
+# Instead of spiloFSGroup, one can define in operator configMap spilo_fsgroup, 
+# automating it for all pg clusters deployments.
+# In Openshift, there is no need to set spiloFSGroup/spilo_fsgroup, as it's being
+# set dynamically at deploytime.
```

Comment:
I think you can remove this part to not make this manifest too long. It's documented nicely in the user docs now.

Reply:
Left only one short note. Rest was removed.